.github/workflows/test-invoke-conda.yml

```diff
@@ -4,29 +4,47 @@ on:
     branches:
       - 'main'
       - 'development'
+  pull_request:
```

Comment:
We don't want to do this. This will run the full suite on every push to a PR branch. The full suite is **extremely** time-consuming and ultimately not necessary anyway, as the existing config runs on *merge* to 'dev'.

The correct fix here would be to have some *minimal* test of some kind, but what we have is already as minimal as possible...

Reply:
So should I remove the pull-request trigger? I mean, than this does not help anybody, if not preventing from merging non working PRs (since there is no package built/deployed/published/you-name it 🙈)

I can remove the PR Trigger 4 sure, but would think about this twice :-o

Reply:
Right, I get what you're saying, and on a normal project I'd agree with you, BUT:

Since the testing takes a REALLY long time, and triggers on *every push*, there are some PRs in here where the full test suite would be running 10s of times, which is not acceptable.

The "insurance" which we *do* have is that testing runs on *merges* to 'development'. That's not a perfect solution, but we've agreed as a project that it's OK for 'development' to be broken at times, as long as we make our best effort to keep it unbroken for the most part.

Reply:
Problem is that it would not work on peoples repositorys unless they set up the secret for the checkpoint themselfs, otherwise it could be done by first checking their branch

Reply:
Right, but we don't care about that, because we don't want to run on PRs

Reply:
So I take it out now, but imho I dont get the point of those Actions when only be running after somehing has been merged (imho it's one step to late for effectivley using those actions 🤷🏻‍♂️)

But np ;P

Reply:
Because on 'development' we can immediately see if something is broken and fix it, and 'development' MUST NOT be merged to 'main' unless it is green, it which case we (should be) guaranteed that 'main' will pass as well.

Reply:
And to not mess it up now: I should remove those lines, right:

```yaml
  pull_request:
    branches:
      - 'main'
      - 'development'
```

Which corresponds to lines 7-10

Reply:
Or do you mean Lines 3-6:

```yaml
  push:
    branches:
      - 'main'
      - 'development'
````

Reply:
Ok, Io am pretty sure you mean 7-10